modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-catalog-impl/src/main/java/com/liferay/headless/commerce/admin/catalog/internal/dto/v1_0/converter/SkuDTOConverter.java

```diff
@@ -57,6 +57,22 @@ public Sku toDTO(DTOConverterContext dtoConverterContext) throws Exception {
 			{
 				cost = cpInstance.getCost();
 				depth = cpInstance.getDepth();
+
+				discontinued = cpInstance.isDiscontinued();
+
+				if (discontinued) {
+					CPInstance discontinuedCPInstance =
```

Comment:
this should be called replacement Sku / somthing like that. 
I don't see the set in the DTO.
Also Brian want to have inline also for this object set with lambdas:
() -> {
    all your if + set the result in the dto
}